lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/view/VertexView.java

```diff
@@ -123,16 +129,20 @@ protected final void layoutChildren() {
         double height = rectangle.getHeight();
 
         double fontWidth = text.getLayoutBounds().getWidth();
-
         text.setFont(Font.font(FONTNAME, (HORIZONTALSCALE) * width / fontWidth));
+        text.setLayoutX(width / 2 - text.getLayoutBounds().getWidth() / 2);
+        text.setLayoutY(height / 2);
+
+        if ((HORIZONTALSCALE * width / fontWidth) < MINTEXTSIZE) {
```

Comment:
Can be refactored into a one-liner: `text.setVisible(HORIZONTALSCALE * width / fontWidth >= MINTEXTSIZE)`.